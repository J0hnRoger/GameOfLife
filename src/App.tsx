import React, { Component } from 'react';

class App extends Component {
  render() {
   return (
    <div className='placehold'>
      <h4 className="placehold__title">App</h4>
    </div>
   );
 }
}

export default App;