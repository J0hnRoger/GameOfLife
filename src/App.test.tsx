import * as React from 'react';
import { shallow } from 'enzyme';

import App from './App';

const app = shallow(<App />);

test('renders correctly', () => {

});