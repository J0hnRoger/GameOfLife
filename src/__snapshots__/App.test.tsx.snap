// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="placehold"
>
  <h4
    className="placehold__title"
  >
    App
  </h4>
</div>
`;
